OpenRA.Mods.Cnc/FileFormats/AudReader.cs

```diff
@@ -59,14 +60,23 @@ public static bool LoadSound(Stream s, out Func<Stream> result, out int sampleRa
 				var dataSize = s.ReadInt32();
 				var outputSize = s.ReadInt32();
 
+				sampleBits = 0;
+				channels = 0;
+
 				var readFlag = s.ReadByte();
 				if (!Enum.IsDefined(typeof(SoundFlags), readFlag))
 					return false;
 
+				sampleBits = readFlag == (int)SoundFlags._8Bit ? 8 : 16;
```

Comment:
```
sampleBits = (readFlag & (int)SoundFlags._16Bit) == 0 ? 8 : 16;
channels = (readFlag & (int)SoundFlags.Stereo) == 0 ? 1 : 2;
```
and then delete the `_8Bit` definition. These are flags, not magic values!

Reply:
That explains why I didn't find that definition in the original file headers.